views/beneficial-owner-individual.html

```diff
@@ -62,9 +62,9 @@
 
         {{ govukInsetText({
           html: "
-            <h2 class='govuk-heading-m'>Information shown on the public register</h2>
-            <p class='govuk-body'>The beneficial owner's home address will not be shown unless it is the same as their correspondence address. Only the month and year of their date of birth will be shown.</p>
-            <p class='govuk-body'>All other information provided in this section will be shown on the public Register of Overseas Entities.</p>
+            <h2 class='govuk-heading-m'>What information we’ll show on the public register</h2>
+            <p class='govuk-body'>We will not show the beneficial owner’s home address unless it is the same as their correspondence address. We’ll only show the month and year of their date of birth.</p>
+            <p class='govuk-body'>We will show all other information you provide on this screen on the public Register of Overseas Entities.</p>
```

Comment:
Can we add a generic mock message that includes this [here](https://github.com/companieshouse/overseas-entities-web/blob/566c59c6c4d351eb5f116ebdc9bfd6cf8c013023/test/__mocks__/text.mock.ts#L91) and add a few simple unit tests please

Reply:
Not sure how that'd work with this.

Reply:
The message `We will show all other information you provide on this screen on the public Register of Overseas Entities` is similar to the due diligence text mock. It should be a generic text message instead of having due diligence on the text name and add unit test to check the presence of the text similar to what has been done on the previous Sam [PR](https://github.com/companieshouse/overseas-entities-web/pull/548)